dist/s_clang_format

```diff
@@ -29,23 +42,16 @@ if [ -n "$is_mongo_repo" ] ; then
 fi
 cd "$dest_dir" || exit 1
 
-# Override existing Clang Format versions in the PATH.
-export PATH="${PWD}/dist":$PATH
-
-# Download the clang-format binary if it's not in place.
-[ ! -x "$(command -v clang-format)" ] && download_clang_format
-
-# Ensure that we have the correct version of clang-format.
+# Check if Clang-Format is already available with the desire version.
 desired_version="10.0.0"
+if ! command -v clang-format &> /dev/null; then
+    download_clang_format $desired_version || exit 1
+elif ! clang-format --version | grep -q $desired_version; then
+    download_clang_format $desired_version || exit 1
+fi
 
-# On macOS Catalina, users need to manually approve binaries.
```

Comment:
It feels this comment is still useful, for folks who start building wiredtiger on macOS.

Reply:
I added it back being more general, see 5af36d4764b48c0291ca3bdf4fb8926dd2065184